csm_web/frontend/src/components/Course.js

```diff
@@ -39,7 +39,7 @@ export default class Course extends React.Component {
     this.state = {
       sections: null,
       sectionsLoaded: false,
-      day: "",
+      dayGroup: "",
```

Comment:
can we just call this `currDayGroup` instead, since that seems to be what it represents?